src/Strategy/ShaStrategy.php

```diff
@@ -16,41 +16,10 @@
 use Humbug\SelfUpdate\Exception\HttpRequestException;
 use Humbug\SelfUpdate\Exception\InvalidArgumentException;
 
-class ShaStrategy implements StrategyInterface
+class ShaStrategy extends ShaStrategyAbstract
```

Comment:
maybe we should add a `@deprecated` tag to the class as well to be able to remove this class in the future